apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -370,17 +370,19 @@ function constructRestrictionRulingPdf(
     })
   }
 
-  doc.text(' ').text(
-    formatMessage(ruling.registratWitness, {
-      registrarNameAndTitle: `${existingCase.registrar?.name ?? '?'} ${
-        existingCase.registrar?.title ?? ''
-      }`,
-    }),
-    {
-      align: 'justify',
-      paragraphGap: 1,
-    },
-  )
+  if (existingCase.registrar) {
```

Comment:
Consider checking for  `existingCase.registrar.name` and `existingCase.registrar.title` here and don't show anything if they are not set. I think `${existingCase.registrar?.name ?? '?'}` is confusing, why would we show "?"

Reply:
I simply removed the "?"s and now, if something is wrong with the user, we will show "undefined" instead of "?". But that will never happen 💣